__tests__/tests.test.js

```diff
@@ -233,3 +233,65 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+  test("PATCH 200; responds with a 200 code and updates an existing article, and responds to user with the updated article", () => {
+    const update = {
+      inc_votes: -20,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(update)
+      .expect(200)
+      .then(({ body }) => {
+        let updatedArticle = body.update;
+        expect(updatedArticle).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          })
+        );
+      });
+  });
+  test("PATCH 404: responds with a 404 status code and returns custom error when the article is a valid query but does not exist", () => {
+    const update = {
+      inc_votes: -20,
+    };
+    return request(app)
+      .patch("/api/articles/9999")
+      .send(update)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("article does not exist");
+      });
+  });
+  test("PATCH 400: responds with a 400 status code and custom bad request error message when the query is invalid", () => {
+    const update = {
+      inc_votes: -20,
+    };
+    return request(app)
+      .patch("/api/articles/not-an-article")
+      .send(update)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("PATCH 400: responds with a 400 status code and custom bad request error message when the vote increment is not a number", () => {
+    const update = {
+      inc_votes: "ham sandwich",
+    };
+    return request(app)
+      .patch("/api/articles/not-an-article")
+      .send(update)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+});
```

Comment:
Perfect tests otherwise! 

Might be worth including 200 - ignores any other (potentially malicious) properties on the object 

Also 400 - inc_votes is missing or spelt wrong Ie {votes: 10}. You could argue it's a slightly different test than 285